CompilerSource/JDI/src/Storage/definition.cpp

```diff
@@ -27,6 +27,8 @@
 #include <cstdio>
 #include <System/builtins.h>
 #include <Parser/handlers/handle_function_impl.h>
+#include <algorithm> // std::max
```

Comment:
Same here, the C headers should go last. Anything `.h` is a C header.

Reply:
again your thinking is backwards

Reply:
C++ headers may include the C headers, not the other way around. Headers likely to be included by other headers go towards the bottom according to Josh/Rusky. Include order goes local headers, distant headers, system/C++ headers, finally followed by C headers.

Reply:
thanks for proving my case you twat. #include <Parser/handlers/handle_function_impl.h> is a local header and <algorithm> is a system header. the rest of the includes are backwards not mine

Reply:
Why is it included using the system header syntax then, I wasn't paying attention to the file name. Can Josh please stop abusing C++?

Anyway: https://stackoverflow.com/a/14243688

Reply:
<> isnt really  system header syntax is just a style choice I applied to the engine but josh never did in jdi. <> and "" do sameshit regarding includes

Reply:
I'll mark as resolved, but yeah that's why I thought it was a C header. Don't get me wrong, it probably is still a C header, just not a standard one.